terraform/main.tf

```diff
@@ -15,17 +15,6 @@ moved {
   to   = aws_s3_bucket.idp_id_broker_search
 }
```

Comment:
You're supposed to leave "moved" blocks in place indefinitely for public code since you can never know when every possible instance has applied the change. 

I'm tempted to remove it anyway, but haven't gotten that bold yet.